src/porepy/models/constitutive_laws.py

```diff
@@ -2181,17 +2183,35 @@ def well_flux_equation(self, interfaces: list[pp.MortarGrid]) -> pp.ad.Operator:
         r_e = self.equivalent_well_radius(subdomains)
 
         f_log = pp.ad.Function(pp.ad.functions.log, "log_function_Piecmann")
-        e_i = self.e_i(subdomains, i=0, dim=9).T  # type: ignore[call-arg]
+
         # We assume isotropic permeability and extract xx component.
+        e_i = self.e_i(subdomains, i=0, dim=9).T  # type: ignore[call-arg]
+
+        # To get a transmissivity, we multiply the permeability with the length of the
+        # well within one cell. For a 0d-2d coupling, this will be the aperture of the
+        # 2d fracture cell; in practice the number can be obtained by multiplying with
+        # the specific volume, or by taking a volume integral over the mortar cell
+        # (which will incorporate the specific volume of the higher-dimensional
+        # neighbor, that is, the fracture). For a 1d-3d coupling, we will need the
+        # length of the well within the 3d cell (see the MRST book, p.128, for comments
+        # regarding deviated wells). Again, this could be obtained by a volume integral
+        # over the mortar cell; however, as 1d-3d couplings have not yet been
+        # implemented, we will raise an error in this case.
+        if any([sd.dim == 3 for sd in subdomains]):
```

Comment:
Should we check that no subdomains are intersections as well? I think something else would break before reaching this code, but better to be explicit about assumptions/cases not yet considered.

Reply:
That would require either calling this function with the wrong set of interfaces, or a very strange misuse of the geometry definition? Unless I have understood your point, I don't think we need to cover those; we generally do not do broad sanity checks of inputs (with good reason).

Reply:
I think at least one of us is misunderstanding/overlooking something. Let's discuss.